xml/System.Windows.Forms/Form.xml

```diff
@@ -6389,10 +6389,11 @@ This example assumes that the `CreateMyOpaqueForm` method is called from another
         <Parameter Name="updateContainerSize" Type="System.Boolean" Index="2" FrameworkAlternate="windowsdesktop-7.0" />
       </Parameters>
       <Docs>
-        <param name="xScaleFactor">To be added.</param>
-        <param name="yScaleFactor">To be added.</param>
-        <param name="updateContainerSize">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="xScaleFactor">The scale factor to be applied on width of the property being scaled.</param>
+        <param name="yScaleFactor">The scale factor to be applied on height of the property being scaled.</param>
+        <param name="updateContainerSize">
```

Comment:
@RussKie Is this parameter named correctly?

Reply:
@dreddy-work

Reply:
That is as intended. @gewarren, do you see any guideline violation here?

Reply:
@dreddy-work I know very little about this method, but I would have expected something more like `updateFormSize` based on the description.

Reply:
thanks @gewarren. It is an override whose base method was defined in the ContainerControl. Form is derived from ContainerControl. If any other ContainerControls override this method, `param` name containing `form` may not suite well.